### DISCUSSION TITLE
New password policy PS8

### DISCUSSION THREAD

**Discussion by ALeclercqWS** [PrestaShop/PrestaShop#36816] [Q&A]
Hello,

I have just migrated from PrestaShop 1.7 to version 8, and I noticed a new password management system that I have implemented. However, I found an inconsistency regarding a variable defined in the theme.yml file of my theme, namely new_password_policy_feature.

I would like to understand how this variable is used, but when I dump it, it returns null. Moreover, I noticed that in the code, especially in the GuestTrackingController file, there are conditions like if variable !== true that indicate the use of the new password management system.

I do not fully understand the purpose of this variable or its actual impact. Could someone please clarify this for me?

Thank you in advance for your help.

![image](https://github.com/user-attachments/assets/7023e5b4-ed41-4746-9abf-8ec7e06be949)

**Comment by matks** [comment]
Hi this setting was introduced here https://github.com/PrestaShop/classic-theme/pull/21/

You can look at the code, if the setting is `true` then some attributes `data-minlength`, `data-maxlength` and `data-minscore` are added, needed for the JavaScript of the password policy to be working.

The setting can be turned false for theme developers who do not want to implement the policy. It's possible, even though not recommended.
